### ISSUE TITLE
Inconsistent use of tabs and spaces

### ISSUE THREAD

**Issue by jmedayil** [sabat54i/photoboothdiy#1]
Hello,

When I try to run this program I receive the prompt - inconsistent use of tabs and spaces indentation. Any advice on how to fix it?

**Comment by amereservant** [comment]
Yes, go through and find all of the lines that it alerts you of, delete the tabs, then replace them with spaces (usually 4 spaces).
